### PR TITLE
[IT] Update `Upgrading to lazer`

### PR DESCRIPTION
**Pull request by marcostudios**
PP on lazer is real 🎉

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Help_centre/Upgrading_to_lazer/it.md [hunk 1]
```diff
@@ -158,35 +151,76 @@ Detto questo, i singoli score e mappe possono essere esportati da lazer e manual
...
 #### Preferisco il vecchio sistema di punteggio, dove il punteggio diventa veramente grande.
 
 Puoi effettivamente modificare l'impostazione `Modalità di visualizzazione del punteggio` in `Classico` per ripristinare lo stile esplosivo del punteggio in tutto il gioco! Non sarà lo stesso, ma ti darà la stessa sensazione del punteggio classico e verrà applicato dove ti aspetteresti.
 
+Anche le classifiche globali useranno il sistema di punteggio classico.
+
 #### Se imposto un punteggio su lazer, rimarrà per sempre?
 
-Anche se cercheremo di preservare il maggior numero possibile di punteggi, **non garantiamo che i punteggi rimarranno a tempo indeterminato**. In qualsiasi momento potremmo decidere di cancellare un sottoinsieme o tutti i punteggi per preservare l'equilibrio del gioco.
+Anche se cercheremo di preservare il maggior numero possibile di punteggi, **non garantiamo che i punteggi rimarranno a tempo indeterminato**. In qualsiasi momento potremmo decidere di cancellare un sottoinsieme di  punteggi per preservare l'equilibrio del gioco, come quando vengono scoperti exploit o comportamenti scorretti.
```
Comment (Sclafus) [inline wiki/Help_centre/Upgrading_to_lazer/it.md:176]:
```suggestion
Anche se cercheremo di preservare il maggior numero possibile di punteggi, **non garantiamo che i punteggi rimarranno a tempo indeterminato**. In qualsiasi momento potremmo decidere di cancellare un sottoinsieme di punteggi per preservare l'equilibrio del gioco, come quando vengono scoperti exploit o comportamenti scorretti.
```

File: wiki/Help_centre/Upgrading_to_lazer/it.md [hunk 2]
```diff
@@ -158,35 +151,76 @@ Detto questo, i singoli score e mappe possono essere esportati da lazer e manual
...
 
 #### I punteggi impostati su stable saranno visualizzati in lazer?
 
-Sì. Una volta terminato il bilanciamento della combinazione di punteggi in lazer e in stable, entrambi saranno visibili.
+Sì.
+
+#### I punteggi impostati in lazer saranno visualizzati in stable?
```
Comment (Sclafus) [inline wiki/Help_centre/Upgrading_to_lazer/it.md:182]:
```suggestion
#### I punteggi fatti su lazer sono visibili su stable?
```

File: wiki/Help_centre/Upgrading_to_lazer/it.md [hunk 3]
```diff
@@ -158,35 +151,76 @@ Detto questo, i singoli score e mappe possono essere esportati da lazer e manual
...
+
+Non al momento.
 
 #### Tutte le mod saranno classificate?
 
-Per ora, i punteggi di tutte le combinazioni di mod appaiono nelle classifiche. Si sta ancora discutendo se i punteggi daranno punti performance con tutte le mod (e se lo faranno, se ci sarà un bonus o una penalità).
+I punteggi di tutte le combinazioni di mod verranno mostrate nelle classifiche.
+
+Però, solo le seguenti mod daranno punti performance per ora:
```
Comment (Sclafus) [inline wiki/Help_centre/Upgrading_to_lazer/it.md:190]:
```suggestion
Però, solo le seguenti mod garantiscono punti performance per ora:
```

File: wiki/Help_centre/Upgrading_to_lazer/it.md [hunk 4]
```diff
@@ -158,35 +151,76 @@ Detto questo, i singoli score e mappe possono essere esportati da lazer e manual
...
+  - Hard Rock (non per osu!mania)
+  - Sudden Death (Configurare `Restart on fail` è consentito)
+  - Perfect (Configurare `Restart on fail` è consentito)
+  - Hidden
+  - Nightcore (solo 1.5x)
+  - Double Time (solo 1.5x, configurare `Adjust pitch` è consentito)
+  - Flashlight
+  - Blinds
+  - Accuracy Challenge
+- Conversione (osu!mania only)
```
Comment (Sclafus) [inline wiki/Help_centre/Upgrading_to_lazer/it.md:207]:
```suggestion
- Conversione (solo osu!mania)
```

File: wiki/Help_centre/Upgrading_to_lazer/it.md [hunk 5]
```diff
@@ -158,35 +151,76 @@ Detto questo, i singoli score e mappe possono essere esportati da lazer e manual
...
+  - Four Keys
+  - Five Keys
+  - Six Keys
+  - Seven Keys
+  - Eight Keys
+  - Nine Keys
+- Divertimento
+  - Muted
+  - No Scope
+- Automazione (osu! only)
```
Comment (Sclafus) [inline wiki/Help_centre/Upgrading_to_lazer/it.md:218]:
```suggestion
- Automazione (solo osu!)
```

**Review by Sclafus — CHANGES_REQUESTED** [review]

**Review by marcostudios — COMMENTED** [review]
Ty :3

**Review by Sclafus — APPROVED** [review]
LGTM :3

**Review by TicClick — APPROVED** [review]

**Pull request merged**